src/pages/Home/Home.js

```diff
@@ -1,25 +1,125 @@
-import { Button } from '../../components';
+import { Button, GameTypes, RulesTextBox } from '../../components';
 import { html } from '../../shared';
+import rickAndMorty from '../../public/img/HomeRickAndMorty.png';
+import styles from './Home.module.css';
 
 /**
  * @param {{} & import('..').RouterProps} props
  * @returns
  */
+
 function Home({ router }) {
-  const pages = [1, 2, 3, 4, 5];
-  return html`<div class="vstack">
-    <h2>Home</h2>
-    ${pages.map((x) =>
-      Button({
-        onClick: (e) => {
-          e.preventDefault();
-          router.goto({ page: 'todo', data: { id: x } });
-        },
-        text: `Go to page ${x}`,
-        variant: 'normal',
-      }),
-    )}
-  </div>`;
+  let selectedCategory = null;
+  let selectedDifficulty = null;
+
+  const rulesBox = RulesTextBox(
+    'Opis poziomu',
+    'What is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing andWhat is Lorem Ipsum? Lorem Ipsum is simply dummy text of the printing and',
+  );
+
+  const btnStart = Button({
+    text: 'Rozpocznij quiz',
+    disabled: true,
+    onClick: () => {
+      router.goto({ page: 'loading', data: { selectedCategory, selectedDifficulty } });
+    },
+  });
+
+  const updateButtonDisabledState = () => {
+    const shouldBeDisabled = selectedCategory === null || selectedDifficulty === null;
+    btnStart.disabled = shouldBeDisabled;
+  };
+
+  const updateRulexTextBox = (header, rules) => {
+    const h3 = document.getElementById('rules').getElementsByTagName('h3')[0];
+    const p = document.getElementById('rules').getElementsByTagName('p')[0];
+    h3.innerHTML = header;
+    p.innerHTML = rules;
```

Comment:
Gdyby `header` i `rules` były zapewniane przez użytkownika mielibyśmy spory problem dotyczący bezpieczeństwa. Nie są, ale dla świętego spokoju i bezpieczeństwa na przyszłość użyłbym `.textContent`.

Reply:
https://youtu.be/06u4AtF-M04 lepiej uważać!